galley/pkg/kube/source/source.go

```diff
@@ -49,6 +53,68 @@ func New(k kube.Interfaces, resyncPeriod time.Duration, cfg *converter.Config) (
 	return newSource(k, resyncPeriod, cfg, kube_meta.Types.All())
 }
 
+// VerifyCRDPresence verifies that all expected CRDs are registered
+// with the k8s kube-apiserver.
+func VerifyCRDPresence(k kube.Interfaces) error {
```

Comment:
Nit: can you place this in a separate file? It is technically not part of the runtime/source model.